apps/sharedtest/test/unit/dialer/font_size_manager_test.js

```diff
@@ -223,6 +223,23 @@ suite('font size manager', function() {
         });
       });
 
+      suite('on <div> with bdi element', function() {
+        setup(function() {
+          view = document.createElement('div');
+          var bdiNode = document.createElement('bdi');
+          bdiNode.textContent = 'testing';
+          view.appendChild(bdiNode);
+          document.body.appendChild(view);
```

Comment:
These DOM mutations are unsafe as we're not sanitizing the DOM after each test. In this case, I would recommend saving and reloading ```document.body.innerHTML``` on ```setup()``` and ```teardown()```, respectively.